pandas/core/groupby/groupby.py

```diff
@@ -1330,7 +1330,9 @@ def any(self, skipna: bool = True):
 
         Returns
         -------
-        bool
+        Series or DataFrame
+            DataFrame or Series of boolean values, which is True if any element is
+            True within each group, False otherwise.
```

Comment:
Should this truthful instead of True?

Reply:
It looks `True` is commonly used in pandas documentation.

https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.any.html
https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.all.html

`True` (capitalized) is not just a noun, but it represents the boolean literal in Python. So I believe `True` is appropriate in this case.

Reply:
Thanks for checking this - agreed.